dist/index.js

```diff
@@ -2168,21 +2172,56 @@ function isAvailable() {
     });
 }
 exports.isAvailable = isAvailable;
-function platforms() {
+function inspect(name) {
     return __awaiter(this, void 0, void 0, function* () {
-        return yield exec.exec(`docker`, ['buildx', 'inspect'], true).then(res => {
+        return yield exec.exec(`docker`, ['buildx', 'inspect', name], true).then(res => {
             if (res.stderr != '' && !res.success) {
                 throw new Error(res.stderr);
             }
-            for (const line of res.stdout.trim().split(`\n`)) {
-                if (line.startsWith('Platforms')) {
-                    return line.replace('Platforms: ', '').replace(/\s/g, '').trim();
+            const builder = {};
+            itlines: for (const line of res.stdout.trim().split(`\n`)) {
```

Comment:
follow up: should add json output (`--format`) for these in build later to avoid this text parsing. Has been asked in buildx repo before as well.

Reply:
Yes indeed would be better to support that directly on buildx.

Reply:
* https://github.com/docker/buildx/issues/325
* https://github.com/docker/buildx/issues/380